tritonbench/operators/flex_attention/operator.py

```diff
@@ -103,32 +124,47 @@ def __init__(
 
         self.batch_size = args.batch
         self.seq_len = args.seq_len
-        self.num_heads = args.n_heads
+        self.num_heads_q = args.n_heads_q
+        self.num_heads_kv = args.n_heads_kv if args.n_heads_kv is not None else args.n_heads_q
         self.head_dim = args.d_head
         self.max_autotune = args.max_autotune
         self.mod_type = args.mod_type
         self.sliding_window_size = args.sliding_window_size
         self.prefix_length = args.prefix_length
+        self.decoding = args.decoding
+        self.dynamic = args.dynamic
 
     def get_input_iter(self) -> Generator:
         """Generate a single input configuration for benchmarking."""
-
+        # Set seed for reproducibility
+        random.seed(42)
```

Comment:
The framework already set the seed at https://github.com/pytorch-labs/tritonbench/blob/main/tritonbench/utils/triton_op.py#L709. The operator should not worry about setting up random seed.